index.html

```diff
@@ -0,0 +1,1379 @@
+          </a>
+          <div class="card-body">
+            <a href="#" class="big-links">
+              LeBron James Says 'Not Only Black People' Deal with Racism in the U.S.
+            </a>
+          </div>
+        </div>
+        <div class="col-lg-4 col-md-12">
+          <div class="row card border-0">
+            <div class="card-body d-inline-block">
+              <a class="float-left mr-2" href="#">
+                <img class="img-fluid" src="./img/nba-national-anthem.webp" alt="doc">
+              </a>
+              <a href="#" class="small-links">
+                  Doc Rivers Says Taking a Knee Was Hardest Part of Lakers vs Clippers Game
+              </a>
+            </div>
+          </div>
+          <div class="row card border-0">
+            <div class="card-body d-inline-block">
+              <a class="float-left mr-2" href="#">
+                <img class="img-fluid" src="./img/nba-national-anthem.webp" alt="nba">
+              </a>
+              <a href="#" class="small-links">
+                Charles Barkley Blasted For Supporting Players Who Stand For Anthem
+              </a>
+            </div>
+          </div>
+          <div class="row card border-0">
+            <div class="card-body d-inline-block">
+              <a class="float-left mr-2" href="#">
+                <img class="img-fluid" src="./img/lebron-james-kawhi-leonard-los-angeles-lakers.webp" alt="kawhi">
+              </a>
+              <a href="#" class="small-links">
+                  NBA Schedule: Clippers vs. Lakers Start Time, TV Channel, Live Stream
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row w-100 d-inline-flex border-top">
+        <div class="col-12 col-lg-7 d-lg-inline-block my-4">
+          <h3 class="red-title">SUBSCRIBE</h3>
+          <div>
+            <img class="float-left mr-5" height="200" src="./img/cover-july-24-2020.jpg" alt="cover">
+            <div class="pt-5">
+              <h4>In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h4>
+              <p>All Access + Weekly Delivery.
+                Choose A Membership That's Perfect for You!</p>
+              <button class="btn-secondary" type="button">SUBSCRIBE</button>
+            </div>
+          </div>
+        </div>
+        <div class="col-12 col-lg-5 my-5 py-2">
+          <h3 class="red-title">NEWSLETTER</h3>
+          <div class="d-inline-flex h-100">
+            <div class="mr-2 d-none d-md-block">
+              <img class="h-100" width="200" src="./img/130387482-newsletter-subscription-button-on-mobile-phone-screen-business-marketing-concept-.jpg" alt="">
+            </div>
+            <div>
+              <h4>The Starting 5</h4>
+              <p>
+                See why nearly a quarter of a million subscribers begin their day with the Starting 5.
```

Comment:
- [ ] This section looks much better. You just need to just the SUBSCRIBE to be equal (the layout), put some border on THE STARTING FIVE, and centralize both.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/89329531-57b8ef00-d665-11ea-9377-6f32fce7a443.png) |![image](https://user-images.githubusercontent.com/61479648/89329597-6bfcec00-d665-11ea-8238-81a047f22549.png)  |